### ISSUE TITLE
Bun hangs on Prisma generate after v1.1.29 on Windows

### ISSUE THREAD

**Issue by i2equalsj2** [oven-sh/bun#14868]
### What version of Bun is running?

1.1.29+6d43b3662

### What platform is your computer?

Microsoft Windows NT 10.0.19045.0 x64

### What steps can reproduce the bug?

Install any version of bun after v1.1.29 (`npm i -g bun@1.1.33`)

The command `bunx --bun prisma generate` outputs the following:

```
Environment variables loaded from .env
Prisma schema loaded from prisma\schema
```

then hangs infinitely. The Prisma client is never generated.

### What is the expected behavior?

In Bun v1.1.29+6d43b3662:

Running `bunx --bun prisma generate` outputs the following:
```
Environment variables loaded from .env
Prisma schema loaded from prisma\schema       

✔ Generated Prisma Client (v5.21.1) to .\node_modules\@prisma\client in 228ms
```
which is correct and expected. The correct Prisma client information is generated and server code works as expected.

**Comment by Jarred-Sumner** [comment]
@190n I worry this is related to the recent napi changes

**Comment by i2equalsj2** [comment]
Issue #5320 is related. 

I noticed there was actually an open issue on Prisma for this: https://github.com/prisma/prisma/issues/19124 which was breaking people's CI/CD pipelines. Using the Prisma `5.21.0-integration-25129.1` branch resolved this error on Windows for me which points to the breakage being parallel moves during Prisma's generate function. I can confirm `bunx --bun prisma generate` is working on Windows and Linux on the new Prisma branch.

TL;DR: 
Using the `5.21.0-integration-25129.1` version of Prisma fixes this error.

`package.json`:
```
"prisma": "5.21.0-integration-25129.1"
```

**Comment by i2equalsj2** [comment]
Update on my previous comment, `bunx --bun prisma generate` still breaks on Windows for me. Prisma v5.22 (which integrates the `5.21.0-integration-25129.1` branch) solves Bun hanging on `bunx --bun prisma generate` for Linux only. I had accidentally run `bunx prisma generate` on Windows which was using my node.js install to finish the prisma generate command.

**Comment by mzdr** [comment]
@190n This seems to be very related, if not a duplicate, of #14536.

**Comment by ben-xiao-io** [comment]
Still an issue on bun 1.2.5

I somehow got it working by installing node directly. Without node, `bunx prisma generate` just hangs
